lib/netsuite/support/actions.rb

```diff
@@ -32,6 +32,8 @@ def action(name)
             self.send(:include, NetSuite::Actions::Search::Support)
           when :add
             self.send(:include, NetSuite::Actions::Add::Support)
+          when :add
+            self.send(:include, NetSuite::Actions::AddList::Support)
```

Comment:
@aom do we know that every record that supports `add` also supports `add_list`? My hunch would be this isn't the case, but I could be wrong here.

If every record does support `add_list` we should eliminate the double `when :add` and add `add` and `add_list` under the same `when`.

Reply:
This was not intentional. I'll change it to `when :add_list`.

AFAIK the addList is widely supported by all record types. At least I can't find any exceptions from NetSuite help section: https://system.eu2.netsuite.com/app/help/helpcenter.nl?fid=section_N3481360.html

Reply:
Ok, good to know! Awesome work on this PR. Excited to get it merged!